content/post/kubedb-v2024.1.31/index.md

```diff
@@ -0,0 +1,537 @@
+
+Here's a sample manifest to provision ElasticsearchDashboard. An Elasticsearch intance `es-cluster` is expected to be provisioned prior to deploying the following manifest.
+
+```yaml
+apiVersion: elasticsearch.kubedb.com/v1alpha1
+kind: ElasticsearchDashboard
+metadata:
+  name: es-cluster-dashboard
+  namespace: demo
+spec:
+  enableSSL: true
+  databaseRef:
+    name: es-cluster
+  terminationPolicy: WipeOut
+```
+
+## Postgres
+
+### Grafana Alerts Dashboard
+In this release, we bring support for Postgres grafana summary dashboard with alerts for a specific Postgres instance.
+
+To use the alert dashboards :
+
+* First you need to have KubeDB operator and a Postgres instance running on you cluster along with Prometheus operator.
+
+* Then create a file named overwrite.yaml having following values. Make sure you change the values matching your needs.
+
+    - For prometheus operator
+        ```yaml
+        resources:
+          - postgres
+
+        dashboard:
+          folderID: 0
+          overwrite: true
+          alerts: true 
+          replacements: []
+
+        grafana:
+          version: “provide your grafana version”
+          url: "provide your grafana service dns" # example: http://grafana.monitoring.svc:80 
+          apikey: "provide grafana apikey"
+
+        app:
+          name: “provide your postgres instance name”
+          namespace: “provide the namespace”
+        ```
+    - For prometheus builtin
+       ```yaml
+        resources:
+          - postgres
+
+        dashboard:
+          folderID: 0
+          overwrite: true
+          alerts: true 
+        replacements:
+          job=\"kube-state-metrics\": job=\"kubernetes-service-endpoints\"
+          job=\"kubelet\": job=\"kubernetes-nodes-cadvisor\"
+          job=\"$app-stats\": job=\"kubedb-databases\"
+
+        grafana:
+          version: “provide your grafana version”
+          url: "provide your grafana service dns" # example: http://grafana.monitoring.svc:80 
+          apikey: "provide grafana apikey"
+
+        app:
+          name: “provide your postgres instance name”
+          namespace: “provide the namespace”
+        ```    
+
+* now follow the steps to install grafana dashboards:
+    ```bash
+    $ helm repo add appscode https://charts.appscode.com/stable/
+    $ helm repo update
+    $ helm search repo appscode/kubedb-grafana-dashboards --version=v2024.1.31
+    $ helm upgrade -i kubedb-grafana-dashboards appscode/kubedb-grafana-dashboards -n kubeops --create-namespace --version=v2024.1.31 
+    -f  /path/to/the/overwrite.yaml
+    ```
+
+### Bug fix and improvements
+* We have fixed a bug in Upgrade OpsRequest for Postgres.
+
+
+## Point in Time Recovery
+
+We have updated the backup directory structure for point-in-time recovery, providing a more organized layout. Going forward, the directories will follow the following standardized structure:
```

Comment:
.... These changes are applicable to all of our archiver supported databases. More specifically : MongoDB, MySQL & Postgres.  ....